packages/stores/src/queries-external/store.ts

```diff
@@ -1,41 +1,28 @@
 import { KVStore } from "@keplr-wallet/common";
-import { HasMapStore, ObservableQuery } from "@keplr-wallet/stores";
 import { DeepReadonly } from "utility-types";
+import { IPriceStore } from "../price";
 import { ObservableQueryPoolFeesMetrics } from "./pool-fees";
-import Axios from "axios";
-
-export class QueriesExternalStore extends HasMapStore<QueriesExternal> {
-  constructor(protected readonly kvStore: KVStore, feeMetricsBaseURL?: string) {
-    super(() => new QueriesExternal(this.kvStore, feeMetricsBaseURL));
-  }
-
-  get(): QueriesExternal {
-    return super.get("external");
-  }
-}
+import { ObservableQueryAccountsPoolRewards } from "./pool-rewards";
 
 /** Root store for queries external to any chain. */
-export class QueriesExternal {
+export class QueriesExternalStore {
   public readonly queryGammPoolFeeMetrics: DeepReadonly<ObservableQueryPoolFeesMetrics>;
+  public readonly queryAccountsPoolRewards: DeepReadonly<ObservableQueryAccountsPoolRewards>;
 
   constructor(
     kvStore: KVStore,
-    feeMetricsBaseURL = "https://api-osmosis.imperator.co"
+    priceStore: IPriceStore,
+    feeMetricsBaseURL = "https://api-osmosis.imperator.co",
+    poolRewardsBaseUrl = "https://api-osmosis-chain.imperator.co"
```

Comment:
How about making these values constant? I've seen these value used in other files

Reply:
Yes, const variable that can be shared